flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -788,4 +896,19 @@ class FlxTilemapExt extends FlxTilemap
 		
 		return value;
 	}
+	
+	private function set_throughDownClouds(Value:Bool):Bool 
```

Comment:
Why are these implemented as `(default, set)` properties? With these setters, a regular variable behaves the same.